ister.py

```diff
@@ -408,43 +411,62 @@ def get_current_format():
         frmt = format_file.read().strip()
     return frmt
 
+def add_default_dnf_config(args):
+    """Supply a default dnf config if one does not exist
+    """
+    os.makedirs(os.path.basename(args.dnf_config), exist_ok=True)
+    with open(args.dnf_config, 'w') as file:
+        file.write("[clear]\nname=clear\nbaseurl=https://download.clearlinux.org/current/x86_64/os/\nenabled=1\n\n[main]\nlogfile=/var/log/dnf.log\n")
 
 def copy_os(args, template, target_dir):
     """Wrapper for running install command
     """
-    LOG.info("Starting swupd. May take several minutes")
-    if not args.format:
-        args.format = get_current_format()
-    add_bundles(template, target_dir)
-    swupd_command = "swupd verify --install --path={0} " \
-                    "--manifest={1}".format(target_dir, template["Version"])
+    package_manager = template["PackageManager"]
+    LOG.info("Starting {0}. May take several minutes".format(package_manager))
+
+    if package_manager == SWUPD:
+        if not args.format:
```

Comment:
Probably would reorganize this section in a different commit.

Reply:
Maybe I could break this out into two different functions?  Something like
```
def install_os(args):
    if package_manager == SWUPD:
        install_os_swupd(args)
    elif package_manager == DNF:
        install_os_dnf(args)
```

Reply:
That'd be fine but I meant the reorganization of the swupd options being appended to the cmd sorry.

Reply:
I'll create a few different commits to organize things a little more nicely as a result of supporting multiple package managers.